<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>649</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="MainDiv_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>651</width>
     <height>601</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>8</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(71, 71, 109);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="Heading">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>50</y>
      <width>290</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
color: rgb(42, 147, 161);
background-color: rgb(255, 255, 255);
}
</string>
    </property>
    <property name="text">
     <string>SEARCH</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="indent">
     <number>2</number>
    </property>
   </widget>
   <widget class="QPushButton" name="SearchButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>470</y>
      <width>90</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-color: rgb(0, 128, 103);
color: rgb(255, 255, 255);
}
QPushButton:hover
{
	
	background-color: rgb(0, 128, 103);
}</string>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QComboBox" name="SelectFilter">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>380</y>
      <width>180</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox
{
	background-color: rgb(225, 60, 94);
color: rgb(255, 255, 255);
}
QComboBox QAbstractItemView
{
	background-color: rgb(0, 128, 103);
    color: rgb(255, 255, 255);
}</string>
    </property>
    <item>
     <property name="text">
      <string>Select Filter</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Contain</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Start With</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>End With</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="SelectColumn">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>180</y>
      <width>180</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox
{
	background-color: rgb(255, 161, 77);
color: rgb(255, 255, 255);
}
QComboBox QAbstractItemView
{
	background-color: rgb(255, 161, 77);
    color: rgb(255, 255, 255);
}</string>
    </property>
    <item>
     <property name="text">
      <string>Select Column</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Name</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Company</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Price</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Reviews</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ratings</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Downloads</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Updated Date</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Category</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="ExitButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>470</y>
      <width>90</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-color: rgb(0, 128, 103);
color: rgb(255, 255, 255);
}
QPushButton:hover
{
	
	background-color: rgb(0, 128, 103);
}</string>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="SearchBar">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>250</y>
      <width>181</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
</string>
    </property>
    <property name="placeholderText">
     <string>Enter Text:</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
